doc/manual/generate-manpage.nix

```
@@ -98,7 +98,7 @@ let
                 (option ? labels)
                 (concatStringsSep " " (map (s: "*${s}*") option.labels));
             in trim ''
-              - `--${name}` ${shortName} ${labels}
+              - <span id="opt-${name}">[`--${name}`](#opt-${name})</span> ${shortName} ${labels}
```

Perhaps best to disambiguate from global options as in `nix.conf`.

```suggestion
              - <span id="cli-opt-${name}">[`--${name}`](#cli-opt-${name})</span> ${shortName} ${labels}
```

---

There should be no overlap whatsoever. Also the global options page for stable commands uses this prefix. I'd opt for shorter anchors rather than namespacing things that don't interfere.

---

```
 $ grep -rl '#opt-' doc/manual/
doc/manual/src/contributing/hacking.md
doc/manual/src/command-ref/nix-build.md
doc/manual/src/command-ref/nix-store/opt-common.md
doc/manual/src/command-ref/env-common.md
doc/manual/src/command-ref/opt-common.md
doc/manual/src/command-ref/nix-channel.md
doc/manual/src/command-ref/nix-collect-garbage.md
doc/manual/redirects.js
```
whereas the conf file uses `#conf-`
```
 $ grep -rl '#conf-' doc/manual/
doc/manual/nix.json
doc/manual/conf-file.json
doc/manual/utils.nix
doc/manual/language.json
doc/manual/utils.nix.gen.hh
doc/manual/generate-builtin-constants.nix
doc/manual/src/advanced-topics/post-build-hook.md
doc/manual/src/glossary.md
doc/manual/src/release-notes/rl-2.17.md
doc/manual/src/release-notes/rl-2.16.md
doc/manual/src/release-notes/rl-2.12.md
doc/manual/src/release-notes/rl-next.md
doc/manual/src/contributing/experimental-feature-descriptions.md
doc/manual/src/contributing/experimental-features.md
doc/manual/src/command-ref/nix-store/realise.md
doc/manual/src/command-ref/env-common.md
doc/manual/src/command-ref/files/profiles.md
doc/manual/src/command-ref/files/default-nix-expression.md
doc/manual/src/command-ref/files/channels.md
doc/manual/src/command-ref/opt-common.md
doc/manual/src/command-ref/conf-file.md
doc/manual/src/command-ref/nix-channel.md
doc/manual/src/command-ref/new-cli/nix3-help-stores.md
doc/manual/src/command-ref/new-cli/nix3-flake.md
doc/manual/src/command-ref/new-cli/nix3-profile.md
doc/manual/src/language/builtin-constants.md
doc/manual/src/language/builtins.md
doc/manual/xp-features.json
doc/manual/redirects.js
```
So there is no conflict.

(Note I am grepping build artifacts and source code alike)